README.md

```diff
@@ -1309,6 +1309,7 @@ _See also [Monitoring](#monitoring)_
 
 - [farmOS](http://farmos.org/) - Web-based farm record keeping application. ([Source Code](https://github.com/farmOS/farmOS)) `GPL-2.0` `PHP`
 - [grocy](https://grocy.info/) - ERP beyond your fridge - grocy is a web-based self-hosted groceries & household management solution for your home. ([Demo](https://en.demo.grocy.info/), [Source Code](https://github.com/grocy/grocy)) `MIT` `PHP`
+- [Grownetics](https://grownetics.co) - Grownetics is a platform to help the world learn how to grow every plant better. Connect sensors to your plants and track them in real time. ([Source Code](https://gitlab.com/grownetics/grownetics)) `GPL-3` `PHP`
```

Comment:
@nodiscc what are your thoughts on this:
The COPYING file is BSD-3-Clause and the LICENSE file is GPL-3

Reply:
>  The COPYING file is BSD-3-Clause and the LICENSE file is GPL-3

Indeed... @NickBusey the license is unclear, can you clarify?